backend/src/main/java/com/example/backend/service/WorkoutService.java

```diff
@@ -33,12 +30,18 @@ public Workout addWorkout(Workout workout) {
         return workoutrepo.save(workoutToAdd);
     }
 
-    public Workout getWorkoutByID(String id){
+    public Workout getWorkoutByID(String id) {
         return workoutrepo.findById(id).orElseThrow(NoSuchElementException::new);
     }
 
     public void deleteWorkoutById(String id) {
```

Comment:
We strongly recomend to avoid not returning anything from deleteWorkoutById(id). You could return the deleted Item or the new list.